### DISCUSSION TITLE
How does ALT+C work or supposed to work?

### DISCUSSION THREAD

**Discussion by AddictArts** [junegunn/fzf#3391] [General]
Also, how does the supposed ESC+C work since ESC quits fzf. Is there a gif of ALT+C working, hard to know exactly what behavior I should expect to see. Thanks for the help.

While I appreciate the WSL support, the mention in the wiki that it is fully functional is not proving correct for me, specifically the keybinding.

**Comment by junegunn** [comment]
You press ALT-C (or ESC-C) in your shell, not while running fzf. It's a key binding for your shell, not one of fzf.



> the mention in the wiki that it is fully functional

The wiki pages are maintained by the community, so I'm not sure which part you're referring to, and I can't vouch for its correctness.

**Comment by AddictArts** [comment]
Thanks @junegunn I appreciate the response.  I don't know how I missed that this is outside `fzf`. The wiki is likely fine, just confusion on how to use these key bindings.  I now have it, thanks again.
